app/models/data_resources/point_of_interest.rb

```diff
@@ -35,6 +35,13 @@ def unique_id
 
     generate_checksum(fields + address_fields + opening_hour_fields.flatten)
   end
+
+  # get travel times for a specific date
+  #
+  # @param [String] date "2023-09-18T12:00"
+  def gtfs_travel_times(date:)
+    PublicTransportation::TravelTime.new(date: date, external_id: external_id, data_provider_id: data_provider_id).travel_times
+  end
 end
 
 # == Schema Information
```

Comment:
ein POI hat Abfahrtszeiten ab einen bestimmten/übergebenem Zeitpunkt.
Diese werden über die Klasse "PublicTransportation::TravelTime" geladen und in einem passenden JSON zurückgegeben